tests/README.md

```diff
@@ -14,17 +14,22 @@ To run these tests, you need to do the following
 
     In general, anything defined in `tests/fixtures/config/env/test.yml`
 
-1. From a terminal, create a virtual environment at `tests/venv`
-   ```ps1
-   & "C:\Program Files\Autodesk\3ds Max {version}\Python\python.exe" -m venv tests/venv
-   ```
+1. From a PowerShell terminal, create a virtual environment at `tests/venv`
+    ```ps1
+    & "C:\Program Files\Autodesk\3ds Max {version}\Python\python.exe" -m venv tests/venv
+    ```
+
+1. Activate the environment
+    ```ps1
+    .\tests\venv\Scripts\Activate.ps1
```

Comment:
Why do you want to activate the environment?

In my experience, this is not necessary if you call the pip binary from inside the env (one less step to do).

Reply:
You're right, it's my old habit of activate it before installing anything.